README.rst

```diff
@@ -68,14 +68,16 @@ Installation
 ------------
 .. readme_installation
 
-To install the latest PyGranta JobQueue release from `PyPI <https://pypi.org/project/ansys-grantami-bomanalytics/>`_,
+To install the latest PyGranta BoM Analytics release from `PyPI <https://pypi.org/project/ansys-grantami-bomanalytics/>`_,
 run this command:
 
 .. code::
 
     pip install ansys-grantami-bomanalytics
 
-To install a release compatible with a specific version of Granta MI, install the
+To install a release compatible with a specific version of Granta MI, consult the
+`Package versions <https://grantami.docs.pyansys.com/version/dev/package_versions.html>`_ section of the metapackage
+documentation, or install the
```

Comment:
I'd lead with the metapackage installation instructions, I think that's still the ideal installation path. So something like:

```
To install a release compatible with a specific version of Granta MI, install the PyGranta metapackage with a requirement specifier:

<code example>

To see which individual PyGranta package versions are installed with each version of the PyGranta metapacakge, consult the `Package versions <link>`_ section of the PyGranta documentation.
```